x-pack/plugins/endpoint/common/types.ts

```diff
@@ -31,9 +31,9 @@ export enum Direction {
 
 export class EndpointAppConstants {
   static BASE_API_URL = '/api/endpoint';
-  static ALERT_INDEX_NAME = 'my-index';
   static ENDPOINT_INDEX_NAME = 'endpoint-agent*';
-  static EVENT_INDEX_NAME = 'endpoint-events-*';
+  static ALERT_INDEX_NAME = 'events-endpoint-1';
```

Comment:
so alerts always go to '-1' and events can go to anywhere starting with 'events-endpoint-'?

Reply:
I have the same question... where do the other events go? Why "-1"?

Reply:
This is because I ran into an issue getting this working with the alert details API.

@madirey @peluja1012 @marshallmain 
The alert details API use the elasticsearch's get API which requires a specific index: https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-get.html the problem with that is I don't think the alert list api returns the specific index that a document is in with the response and the event index will roll over based on the ILM policy. So it'll change from `events-endpoint-000001` to `events-endpoint-000002` etc. I tried having the alert details request take a pattern like `events-endpoint-*` but that causes a 500 from elasticsearch because the get API needs a single index.

I think we need to encode the index of the document in the `_id` or something so when the frontend passes it back, the backend to determine the specific index since we won't know that ahead of time.

Another example:
![image](https://user-images.githubusercontent.com/56361221/75712908-c51bba80-5c96-11ea-84ec-f5d5acb34b36.png)

Reply:
The index is returned by ES for each hit when searching... we can return that in the response to the FE and/or cache it... will need some logic to look it up if it's not available, or we could just change it to use the search API...

Reply:
Is this a case where we should use aliases for the alert indexes to unify them into a single index endpoint?

Reply:
@madirey we could probably just base64 encode the index and document _id together into the `_id` field that is in the alert details response right? That way it's opaque to the frontend?

We should probably avoid the search api just because I would think it'd have a little more overhead

Reply:
@achuguy hmm I don't think we could use an alias to abstract the search/get api. The endpoint will use an alias for all events in general so it doesn't know the specific rollover index either. We could use a pipeline to break them out but we already made the decision to write the alerts and events to a single index.

Reply:
@jonathan-buttner Yeah, that sounds like a good approach...

Reply:
@jonathan-buttner If the endpoint is using an alias to write to, can you use a get with the same alias or are you specifically trying to isolate you get to a single index?

Reply:
@achuguy so I believe you can have an alias that points to multiple indices but it doesn't look like you can do a `GET` on an alias with multiple indices:

![image](https://user-images.githubusercontent.com/56361221/75784248-ee3c5980-5d2f-11ea-8380-f089559dbe36.png)

You can execute searches I believe though but I think we should avoid doing a search since we have all the info we need to just run a `GET`.

Reply:
Ah, thanks for clearing that up for me